yktrace/src/lib.rs

```diff
@@ -218,6 +240,9 @@ mod test_helpers {
         assert!(trace.into_iter().count() < trace.raw_len());
     }
 
+    // FIXME Fails because it fails to see the trace tail.
+    // This is presumably an issue with the mapper since the labelling scheme changed for new SIR.
```

Comment:
This sounds a bit bad?

Reply:
It's the next unit of work for @ptersilie ;)

Reply:
Lucky him :)